pages/code_schools.js

```diff
@@ -6,19 +6,23 @@ import FlatCard from 'components/Cards/FlatCard/FlatCard';
 import SchoolCard from 'components/Cards/SchoolCard/SchoolCard';
 import Button from 'components/Button/Button';
 import OutboundLink from 'components/OutboundLink/OutboundLink';
+import Modal from 'components/Modal/Modal';
 import { getCodeSchoolsPromise } from 'common/constants/api';
 import States from 'common/constants/dropdown-states-values';
 import edx from 'static/images/moocs/edx.jpg';
 import treehouse from 'static/images/moocs/treehouse.jpg';
 import udacity from 'static/images/moocs/udacity.jpg';
 import styles from './styles/code_schools.css';
 
+const INITIAL_MODAL_STATE = { name: '', locations: [] };
+
 export default class CodeSchools extends React.Component {
   state = {
     allSchools: [],
     filteredSchools: [],
     moocSchools: [],
     selectedStates: [],
+    locationsModalInfo: INITIAL_MODAL_STATE,
```

Comment:
I think I'd just prefer `{ name: '', locations: [] }` used directly/inline here _and_ the reset function handler. No need to put it into a variable.